ai-feature-pack/src/main/resources/modules/system/layers/base/dev/langchain4j/main/module.xml

```diff
@@ -22,6 +24,7 @@
     <dependencies>
         <module name="com.google.code.gson"/>
         <module name="org.slf4j"/>
+        <module name="java.net.http"/>
         <module name="java.logging"/>
         <module name="com.squareup.retrofit2" export="true" services="import"/>
```

Comment:
We usually keep these sorted the same way as we do import ordering.

Reply:
yes, sorry thought http not net.http ;)